src/command_modules/azure-cli-acr/azure/cli/command_modules/acr/_params.py

```diff
@@ -42,9 +41,4 @@
 
 register_cli_argument('acr create', 'registry_name', completer=None)
 register_cli_argument('acr create', 'resource_group_name',
-                      validator=validate_resource_group_name)
-
-register_cli_argument('acr', 'enable_admin', help='Enable admin user',
-                      validator=validate_admin_user)
-register_cli_argument('acr', 'disable_admin', help='Disable admin user',
-                      validator=validate_admin_user)
+                      validator=validate_resource_group_name)
```

Comment:
I was not able to run pylint on the solution. Visual Studio crashes every time I try to run pylint from the solution explorer. Is there another way to run pylint? I tried `pylint /` but there was no __init__.py in the root.